<launch>
  <arg name="rviz" default="true" />

  <node pkg="obsc" type="obj_detection_node" name="obj_detection_node" output="screen" respawn="true">
  </node>
  <!--param name = "left_rec" type="int" value = "4"/-->
  <!--param name = "right_rec" type="int" value = "-2"/-->
  <group if="$(arg rviz)">
  <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find obsc)/rviz/obsc.rviz" />
  </group>
 
</launch>
